docs/docs-content/troubleshooting/edge/collect-support-bundles.md

```diff
@@ -0,0 +1,149 @@
+---
+sidebar_label: "Collect Support Bundles"
+title: "Collect Support Bundles for Edge Cluster Troubleshooting"
+description: "Learn how to deploy a Private Cloud Gateway (PCG) to an existing Kubernetes cluster."
+hide_table_of_contents: false
+sidebar_position: 1
+toc_min_heading_level: 2
+toc_max_heading_level: 3
+tags: ["edge", "troubleshooting", "support"]
+---
+
+When deploying or operating Edge clusters, you may encounter issues that require assistance from the Spectro Cloud
+support team, such as Edge hosts failing to register, pod failures, or provisioning errors. To help our team
+troubleshoot these issues more effectively, follow this guide to collect support bundles and attach them to your support
+ticket.
+
+## Limitations
+
+- This guide applies only to connected installation mode, not airgap environments, since it requires internet access to
```

Comment:
Remember that we no longer use "connected vs airgap", we now use "centrally vs locally managed". Though maybe you are using this language only because this one is getting backported. 

Also, I would remove the limitation and put this as a prerequisite that the cluster has a connection to the internet. Even if your cluster is locally managed, I don't think there is a reason that this script wouldn't work as long as it has an internet connection.